src/index.ts

```diff
@@ -68,13 +83,44 @@ export class MaticPOSClient extends SDKClient {
     return this.posRootChainManager.exitERC20(txHash, options)
   }
 
+  exitERC20Token(txHash: string, options?: SendOptions) {
+    if (!txHash) {
+      throw new Error(`txHash not provided`)
+    }
+    if (options && !options.from) {
+      throw new Error(`options.from is missing`)
+    }
+    return this.posRootChainManager.exitERC20Hermoine(txHash, options)
+  }
+
```

Comment:
Use old exitERC20 method with `fastProof: boolean` flag in options. Use hermione if this flag is true, else use old logic. This way we can have same interface for both implementations and upgrade in future to use hermione by default without any changes on dapp side.